src/spikeinterface/qualitymetrics/quality_metric_calculator.py

```diff
@@ -91,6 +91,14 @@ def _run(self, verbose, **job_kwargs):
         progress_bar = job_kwargs["progress_bar"]
 
         unit_ids = self.sorting.unit_ids
+        non_empty_unit_ids = self.sorting.get_non_empty_unit_ids()
+        empty_unit_ids = self.sorting.get_empty_unit_ids()
+        if len(empty_unit_ids) > 0:
+            warnings.warn(
+                f"Units {empty_unit_ids} are empty. Quality metrcs will be set to NaN "
+                f"for these units.\n To remove empty units, use `sorting.remove_empty_units()`."
+            )
+
```

Comment:
This will be quite annoying I think repeat over time.
Putting nan is enough no ?
Or we can add a column in the dataframe with a flag empty units ? An empty units is also some kind af a "metrics" we can use later for removal!

Reply:
This will only be printed once! And I think it's good to know! Maybe the user has forgotten to remove them...

Reply:
I think it's handy to have this information available in the log as it might not be initially clear why there are NaN in the quality metrics output, and checking the log will save looking through the code. It will only shown in the case that the automatic removal of empty units is set `False` so should not show in most cases anway.

Reply:
Agreed! Let's keep it